src/pipelines/epidemiology/us_ny_authority.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+from typing import Any, Dict, List
+import numpy
+from pandas import DataFrame, concat, merge
+from lib.cast import safe_float_cast
+from lib.io import read_file
+from lib.pipeline import DataSource
+from lib.time import datetime_isoformat
+from lib.utils import table_multimerge, table_rename
+
+
+def _parse_boro(data: DataFrame, column_prefix: str, fips: str) -> DataFrame:
+    data = table_rename(
```

Comment:
Might be worth documenting why this renaming is necessary. Is it to conform to some convention? Is it to be consistent with other output data?

Reply:
It's how the data parsing works at the fundamental level. The `parse` (or `parse_dataframes`) function needs to output a `DataFrame` with column names corresponding to the specific `DataPipeline` `config.yaml` file.